scripts/find_outliers.py

```diff
@@ -22,6 +26,196 @@ def find_outliers(data_directory):
     None
     """
     # Your code here
+
+    ## MLN:  10-5-16
+
+    #make an empty string array that will ultimately store which volumes
+    #are outliers
+    outlierarr = []
+
+    # Identify image files in data_directory using names listed in
+    # 'hash_list.txt' (executed similarly to validate_data.py)
+    hashlist = str(data_directory + '/' + 'hash_list.txt')
+
+    #iterating through to add to an array which contains the values for each volume for each files
+    k = 0
+    vol_means = {}
+    volumes = {}
+    imgfiles = {}
+
+    for line in open(hashlist, 'rt'):
+        i = line.split()
+        imgfile = str(data_directory + '/' + i[1]) #making this filename a string with full path
+        #print(imgfile) #testing to see through printing whether the data reads in
+        # Load the image file
+        img = nib.load(imgfile, mmap=False)
+        # Retrieve data from image array
+        data = img.get_data()
+
+        volumes[str(k)] = data
```

Comment:
I guess you could also make `volumes` a list, and append.  Or you could use the filename as a key to the dictionary, as in `volumes[i[1]] = data`.  You don't need the `str` here, dictionaries also allow numbers as keys.  A better variable name than `k` would also help, maybe `img_no`.